qlty-check/src/executor.rs

```diff
@@ -692,21 +692,9 @@ fn run_invocation(
                 return;
             }
 
-            file_result.issues = file_result
-                .issues
-                .par_iter()
-                .cloned()
-                .filter_map(|mut issue| {
-                    for transformer in transformers {
-                        if let Some(transformed_issue) = transformer.transform(issue) {
-                            issue = transformed_issue;
-                        } else {
-                            return None;
-                        }
-                    }
-                    Some(issue)
-                })
-                .collect();
+            for transformer in transformers {
+                file_result.issues = transformer.transform_batch(file_result.issues.clone());
```

Comment:
Is the clone() needed here?

Reply:
Technically in this usage it is. We could modify a bit to avoid the clone, but I'm unclear if this actually avoids "cloning".